docs/configuration/thumbnails.md

```diff
@@ -39,5 +40,42 @@ Settings
 | `save_files` | Can be either `true` or `false`. If true, it saves generated thumbnails to disk so they don't need to be generated again. |
 | `allow_upscale` | Can be either `true` or `false`. If true, the thumbnailer can make images bigger than the original. |
 | `exif_orientation` | Can be either `true` or `false`. If true, the thumbnailer will use the image orientation from its EXIF data. |
+| `only_aliases` | Can be either `true` or `false`. If true all thumbnail requests which don't belong to an alias will be blocked |
 | `browser_cache_time` | Sets the amount of seconds that the browser will cache images for. Set it to activate browser caching. |
 
+Thumbnail Aliases
+-----------------
+
+With thumbnail aliases, you can define thumbnail sizes which are specific for your theme, 
+like a `teaser` or `cover` thumbnail. This not only makes them easy to change at any time,
+but also prevents people from flooding your cache or server space with automatically generated
+thumbnails.
+
+### Defining Thumbnail Aliases
+
+Because thumbnail aliases are very theme specific, 
+they will be defined in the `theme.yml` of your theme.
+
+Here is a little example with a `teaser` and a `cover` alias:
+
+```yaml
+thumbnails:
+    aliases:
+        teaser:
+            size: [400,300]
+            cropping: crop 
+        cover:
+            size: [600,400]
+            cropping: crop 
+```
+
+As you can see, each alias has a new setting called `size` where you define the width and the height
+of the thumbnails. You can also define how they will be cropped.
+
+### Preventing non-alias Thumbnails
+
+With the `only_aliases` setting in the general thumbnail configuration, you can prevent the generation
+of thumbnails that don't belong to an alias. 
+
+This is useful to secure your server from automatically
+generated thumbnails that could flood your cache.
```

Comment:
Might want to check which is 80 character lines, and which are shorter/longer … I would … but lunch! :wink: